src/main/java/project/Application.java

```diff
@@ -0,0 +1,22 @@
+package project;
+
+import org.springframework.boot.orm.jpa.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+/**
+ * Created by slava23 on 10/11/2016.
+ */
+
+@SpringBootApplication
+@EnableJpaRepositories(basePackages = {"project/model"})
```

Comment:
why "project"? I don't see this folder in source code.
@ComponentScan is a part of @SpringBootApplication (http://docs.spring.io/autorepo/docs/spring-boot/current/reference/html/using-boot-using-springbootapplication-annotation.html)
Also, check, maybe @EntityScan is also not needed.
Yes, seems like not needed because of http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-using-the-default-package